oneview_redfish_toolkit/app.py

```diff
@@ -66,6 +69,68 @@
 app.register_blueprint(thermal)
 app.register_blueprint(storage_collection)
 
+
+@app.errorhandler(status.HTTP_400_BAD_REQUEST)
+def bad_request(error):
+    """Creates a Bad Request Error response"""
+    redfish_error = RedfishError(
+        "PropertyValueNotInList", error.description)
+
+    redfish_error.add_extended_info(
+        message_id="PropertyValueNotInList",
+        message_args=["VALUE", "PROPERTY"],
+        related_properties=["PROPERTY"])
+
+    error_str = redfish_error.serialize()
+    return Response(
+        response=error_str,
+        status=status.HTTP_400_BAD_REQUEST,
+        mimetype='application/json')
+
+
+@app.errorhandler(status.HTTP_404_NOT_FOUND)
+def not_found(error):
+    """Creates a Not Found Error response"""
+    redfish_error = RedfishError(
+        "GeneralError", error.description)
```

Comment:
Is it really a "GeneralError"?  Its really a not found error, Redfish doesn't have a type for a not found error?

Reply:
no it does not. I had a meeting with the PO today because of this and what he defined I wrote in the issue 80 comments, Please take a look

Reply:
Ok sounds perfect.